pkg/azuredisk/azuredisk.go

```diff
@@ -51,8 +51,9 @@ const (
 	defaultDiskMBpsReadWrite = 100
 
 	// see https://docs.microsoft.com/en-us/rest/api/compute/disks/createorupdate#uri-parameters
-	diskNameMinLength = 1
-	diskNameMaxLength = 80
+	// see https://docs.microsoft.com/en-us/rest/api/compute/snapshots/createorupdate#uri-parameters
+	diskOrSnapshotNameMinLength = 1
```

Comment:
keep `diskNameMinLength` and add comment and link that snapshot naming rule is the same